### PR TITLE
Update HomePage.js

### PR DESCRIPTION
**Pull request by Dinnall**
### What's this PR do?

This pull request replaces the static sponsor's section on the homepage.
With this change, sponsors can be easily added to contentful, then pulled from GraphQL

### How should this be reviewed?

...

### Any background context you want to provide?

...
Screen Shot 2021-02-22 at 9 38 40 PM

### Relevant tickets

References [Pivotal #176745890](https://www.pivotaltracker.com/n/projects/2401401/stories/176745890).

### Checklist

- [  x  ] This PR has been added to the relevant Pivotal card.
- [  x ] Added screenshots of front-end changes on small, medium, and large screens.

### CODE REVIEW

File: resources/assets/components/pages/HomePage/HomePage.js [hunk 1]
```diff
@@ -5,7 +5,6 @@ import { css } from '@emotion/core';
 import { React, Fragment } from 'react';
 
 import PageQuery from '../PageQuery';
-import sponsorList from './sponsor-list';
```
Comment (mendelB) [inline resources/assets/components/pages/HomePage/HomePage.js:8]: Can we delete this file now, or are we using this list elsewhere?

**Review by mendelB — APPROVED** [review]
🔥

**Comment by DFurnes** [comment]
So excited about this change! Before we merge, let's make sure we set up the production homepage's Contentful entry with the correct list of sponsors. After that, you can refresh our "QA" Contentful environment by running [this Jenkins job](https://jenkins.dosomething.org/job/QA%20Refresh%20-%20Part%204%20(Contentful)/) (via the "Build Now" button in the sidebar) to pull those changes over to [qa.dosomething.org](https://qa.dosomething.org) for pre-deploy testing.

**Comment by weerd** [comment]
Exciting! I didn't realize this was happening! Glad it's not hardcoded anymore 💯  Nice work @Dinnall 👍

**Comment by Dinnall** [comment]
> qa.dosomething.org

I just ran Jenkins. Going to merge now

**Pull request merged**